plugins/module_utils/redfish_utils.py

```diff
@@ -410,9 +409,6 @@ def _get_extended_message(error):
                 pass
         return msg, data
 
-    def _init_session(self):
-        pass
```

Comment:
Some code in other collections might depend on this. (I think there's at least one other collection that uses redfish_utils from community.general, maybe something from DELL? Maybe someone else has a better recollection :) )

Reply:
True, but Ansible's code is riddled with "private" attributes that are used everywhere :) We should properly deprecate this and remove it in the next major version.

Reply:
For what it's worth, I don't see anyone overriding this method today (unless they're doing so outside of modules in this repository).

I'm inclined to remove it as well since we have a command to create a session (`CreateSession` in the `Sessions` category in redfish_command.py), which calls `create_session` in redfish_utils.py. The response data provides the session token, which is passed in subsequent commands with the `auth_token` parameter.

Reply:
@mraineri while I tend to prefer getting rid of useless stuff right away, if some other collection makes reference to this, then it becomes a breaking change. Anyways, that codes has been sitting there for years, it can wait a tad longer to go away.

Added the deprecation.